File: Java/GCD.java
import java.util.Scanner;
public class main{
    public static int gcd(int a,int b){
       if(b==0)
          return a;
       else
          return gcd(b,a%b);
    }
    public static void main(String[] args){
    	 Scanner input = new Scanner(System.in);
    	 System.out.println("Enter 1st number/-");
    	 int a = input.nextInt();
    	 System.out.println("Enter 2nd number/-");
         int b = input.nextInt();
         int GCD = gcd(a,b);
         System.out.println("GCD is: "+GCD);
    }
}

File: piyal/Python/GCD.py
def compute_gcd(x, y):
    smaller = min(x,y)
    gcd = 1
    for i in range(1, smaller+1):
        if((x % i == 0) and (y % i == 0)):
            gcd = i
    return gcd

num1 = int(input("Enter first number/-"))
num2 = int(input("Enter second number/-"))

gcd = compute_gcd(num1, num2)

print("GCD of", num1,"and", num2,"is", gcd)